test/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>VideoJS Tests</title>
+  <link rel="stylesheet" href="../node_modules/qunitjs/qunit/qunit.css">
+</head>
+<body>
+  <div id="qunit"></div>
+  <div id="qunit-fixture"></div>
```

Comment:
If the globals-shim is used with this html, we should probably delete this fixture because it will be duplicated by the shim.